Formula/reminiscence.rb

```diff
@@ -22,8 +22,8 @@ class Reminiscence < Formula
   uses_from_macos "zlib"
 
   resource "tremor" do
-    url "https://git.xiph.org/tremor.git",
-        :revision => "7c30a66346199f3f09017a09567c6c8a3a0eedc8"
+    url "https://gitlab.xiph.org/xiph/tremor.git",
+        :revision => "b56ffce0c0773ec5ca04c466bc00b1bbcaf65aef"
```

Comment:
This one is actually older and contains security vulnerabilities. Seems like they hadn't completely mirrored tremor to GitLab.

Reply:
I found another source for that particular revision.

Reply:
Looks good. `git.xiph.org` will return eventually so can easily use that again for `tremor` in the future.